src/custom/pages/About/index.tsx

```diff
@@ -28,6 +31,10 @@ const Wrapper = styled(Page)`
 export default function About() {
   return (
     <Wrapper>
+      <Helmet>
+        <title>About - {APP_TITLE}</title>
+      </Helmet>
```

Comment:
May I suggest you make this a component that takes as an optional parameter the actual page title?

Something like:

```ts
function PageTitle(customTitle?: string) {
  return <Helmet><title>{customTitle ? `${customTitle} - ` : ''}{APP_TITLE}</title></Helmet>
}
```